modules/api/pom.xml

```diff
@@ -62,6 +62,16 @@
             <artifactId>hamcrest-optional</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.tngtech.archunit</groupId>
+            <artifactId>archunit</artifactId>
```

Comment:
`test` scope is missing

Reply:
It is set in parent pom:
https://github.com/apache/ignite-3/blob/0dc1b5852531c2e7f06dc1091c21b94c454fedd3/parent/pom.xml#L802